test/compilable/extra-files/vcg-ast.d.cg

```diff
@@ -0,0 +1,159 @@
+=== compilable$?:windows=\|/$vcg-ast.d.cg
```

Comment:
This is a temporary workaround until i improve the path handling with `TEST_OUTPUT_FILE`

Reply:
This seems to creep-up in most PR. Can we get the fix before the transition and avoid the workarounds alltogether ?

Reply:
Working on it, I just wanted to get a better grasp on the correct approach before committing to a solution.

Reply:
Added affd594, see https://github.com/dlang/dmd/pull/10922#discussion_r659171570 for a discussion